lib/chewy/type/crutch.rb

```diff
@@ -23,7 +24,11 @@ def #{name}
 
       module ClassMethods
         def crutch name, &block
-          self._crutches = _crutches.merge(name.to_sym => block)
+          if block
+            self._crutches = _crutches.merge(name.to_sym => block)
+          else
+            self._crutches_classes = _crutches_classes + [name]
```

Comment:
Yea, I had the comment somewhere. I like this design more that the one which is proposed in the current PR:

https://github.com/toptal/platform/pull/9800#issuecomment-188677166

It is completely backwards compatible with the current interface of crutches, it's footprint is very small (the diff is gonna be short), it can be extended to using named crutches, and there are a bunch of other advantages described in the comment I mentioned.

> The only thing to keep in mind here: all the block crutch definitions will go to the first crutch argument in the value proc, blockless classes provided will be instantiated additionaly and passed as following arguments.

This ^ is kinda too hard to understand.